lms/djangoapps/certificates/views/support.py

```diff
@@ -90,6 +90,7 @@ def search_by_user(request):
         cert["course_key"] = unicode(cert["course_key"])
         cert["created"] = cert["created"].isoformat()
         cert["modified"] = cert["modified"].isoformat()
+        cert["download_url"] = cert["download_url"] or api.get_certificate_url(user.id, cert["course_key"])
```

Comment:
@saleem-latif instead of manipulating `download_url` here I would suggest to do it inside `get_certificates_for_user` API.